apps/web-app/components/settings/addTotp.vue

```diff
@@ -37,7 +37,7 @@
 
   // creation step
   const qrUri = ref('');
-  const qrCode = useQRCode(qrUri, { width: 200, height: 200, margin: 5 });
+  const qrCode = useQRCode(qrUri, { width: 200, margin: 5 });
```

Comment:
@BlankParticle why was the height removed?
we need it to be square, there was a time when it didnt render properly